README.md

```diff
@@ -62,7 +62,7 @@ And add this line to your welcome view:
 
      <%= link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path %>
 
-If you visit your page and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
+If you visit your page(`users/sign_in`) and click "Sign in with Facebook," you should be able to go through a Facebook application authorization flow.
```

Comment:
@gj @mendelB Yeah, I am also not sure this change is correct. It's been so long ago since I made it, I forget what was confusing to me. I just had a quick look at my old lab and the page I was using was, indeed, the welcome page. I remember being confused about something at that stage though, so apparently...I was confused. Perhaps, it's best to just be more explicit about what 'your page' really means.